templates/alarms/content/alarms.tf

```
@@ -11,8 +11,8 @@ resource "aws_cloudwatch_log_metric_filter" "filter_system_error" {
 }
 
 resource "aws_cloudwatch_metric_alarm" "alarm_system_error" {
-  alarm_name          = ${{ values.error_alarm_name | dump}}
-  alarm_description   = ${{ values.error_alarm_description | dump }} 
+  alarm_name          = ${{ values.service_name}}-Errors
+  alarm_description   = ${{ values.service_name }} errors >= ${{ values.error_threshold }} in 1 minute
```

Just a note that you'll need quotes around these now:
```suggestion
  alarm_name          = "${{ values.service_name}}-Errors"
  alarm_description   = "${{ values.service_name }} errors >= ${{ values.error_threshold }} in 1 minute"
```